proof-templates.json

```diff
@@ -121,10 +105,6 @@
                         "family_name"
                      ],
                      "restrictions": [
-                        {
```

Comment:
Is this a change to the Full name proof? Was there an ask to remove the schema and issuer restrictions?